test/k8sT/Services.go

```diff
@@ -1642,14 +1642,27 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 
 			terminateServiceEndpointPod()
 
+			By("Checking if client pod terminated gracefully")
+			ctx, cancel = context.WithCancel(context.Background())
+			res = kubectl.LogsStream(helpers.DefaultNamespace, clientPod, ctx)
+			// The log message indicates that the connectivity between client and
+			// server was intact even after the service endpoint pod was terminated,
+			// and that the client connection terminated gracefully.
+			Expect(res.WaitUntilMatch("exiting on graceful termination")).To(BeNil(),
+				"%s is not in the output after timeout", res.GetStdOut())
```

Comment:
On a different topic, I think the error message is incorrect here. It will end up being:
```
[pod logs...] is not in the output after timeout
```
Did you mean for the error message to be something like follows?
```
'exiting on graceful termination' not found in logs after timeout: [pod logs...]
```

Reply:
Fixed.